pkg/metrics/chaos-controller-manager.go

```diff
@@ -19,6 +19,8 @@ import (
 	"context"
 	"reflect"
 
+	z "github.com/chaos-mesh/chaos-mesh/pkg/log"
```

Comment:
please do not use an alias `z` which does not make sense to developers. IMO, importing it without an alias is OK for me .